src/@dvcorg/gatsby-theme-iterative/getEditLink.ts

```diff
@@ -0,0 +1,3 @@
+const getEditLink = (sourcePath: string): string =>
+  `https://github.com/iterative/dvc.org/blob/master/content${sourcePath}`
+export default getEditLink
```

Comment:
This is what broke edit links last time. `constants` logic was changed to a simple function override- does the same thing, adaptable to any format we want, practically same amount of effort for changes.